amundsen_application/static/css/_variables-default.scss

```diff
@@ -128,6 +128,7 @@ $resource-title-color: $indigo60;
 
 // Spacing
 $spacer-size: 8px;
+$spacer-half: $spacer-size/2;
```

Comment:
is $spacer-size/2 the convention or $spacer-size / 2?

Reply:
I would say whatever stylelint does, if it doesn't care, we neither!